CONTRIBUTING.md

```diff
@@ -26,11 +26,17 @@ The process to release a new version is:
 1. Update the version in `VERSION` and run `make VERSION.h`
 2. Update the Debian changelog with `dch -v {new version}`.
 3. Update the two `wget` urls in the README to point to the new version.
-4. Run `make release`
-5. Commit the changes and tag the commit like `v1.0.0`.
-6. `git push --tags origin master`
-7. Run `twine upload --skip-existing dist/*.tar.gz dist/*.whl` to upload the
+4. Push these changes as a new branch (has to be on Yelp/dumb-init, not a fork)
+   and let Travis build it.
+5. Find and download the binary and Debian packages for both amd64 and ppc64el;
+   there will be links printed at the end of the Travis output. Put these into
+   your `dist` directory.
+6. Run `make release`
+7. Put your commit on master (no merge or PR needed) and tag the commit like
+   `v1.0.0`.
+8. `git push --tags origin master`
+9. Run `twine upload --skip-existing dist/*.tar.gz dist/*.whl` to upload the
    new version to PyPI
-8. Upload the resulting Debian package, binary (inside the `dist` directory),
+10. Upload the resulting Debian package, binary (inside the `dist` directory),
```

Comment:
might be easier to make the commit first, push it to master, then upload the artifacts (would save the "make a separate branch" dance)

(this is roughly how I do the release for https://github.com/sass/libsass-python btw)

Reply:
true, that's probably easier